straxen/plugins/peaklet_processing.py

```diff
@@ -535,20 +535,23 @@ class PeakletClassificationHighEnergy(PeakletClassification):
 
     def compute(self, peaklets_he):
         return super().compute(peaklets_he)
-    
+
 
 FAKE_MERGED_S2_TYPE = -42
 
 
 @export
 @strax.takes_config(
-    strax.Option('s2_merge_max_area', default=5000.,
-                 help="Merge peaklet cluster only if area < this [PE]"),
-    strax.Option('s2_merge_max_gap', default=10_000,
-                 help="Maximum separation between peaklets to allow merging [ns]"),
     strax.Option('s2_merge_max_duration', default=50_000,
                  help="Do not merge peaklets at all if the result would be a peak "
-                      "longer than this [ns]"))
+                      "longer than this [ns]"),
+    strax.Option('s2_merge_gap_thresholds', default=((1.7, 2.65e4), (4.0, 2.6e3), (5.0, 0.)),
```

Comment:
Hi Tianyu, thanks for the nice PR. I think it's good to merge this for the next reprocessing, but can you explain how you decided these values? From the plots [here](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:tzhu:straxencalibrateclustering#current_status), it looks bit different.

Reply:
Hi Shingo, I did redo [that plot](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:tzhu:straxenmerges2s#merge_thresholds) with simulation, and think that we can tolerate 1% loss of some long and small S2 peaks near the cathode.

Reply:
Thanks Tianyu for the updated note. It looks really promising!! I agree that 1% loss should be fine (around 3000 pe, it might be ~5% loss?).
I think you are still investigating, but it's interesting to see where the population " XX events found in default is now missing" went to.

Reply:
As you directly explained to me, this requirement is conservative considering that parameters are estimated using cathode events. Then, I'm fine with these numbers